Makefile

```diff
@@ -1482,6 +1488,60 @@ $(GOVULNCHECK): # Build govulncheck.
 $(IMPORT_BOSS): # Build import-boss
 	GOBIN=$(TOOLS_BIN_DIR) $(GO_INSTALL) $(IMPORT_BOSS_PKG) $(IMPORT_BOSS_BIN) $(IMPORT_BOSS_VER)
 
+## --------------------------------------
+## triage-party
+## --------------------------------------
+
+.PHONY: release-triage-party
+release-triage-party: docker-build-triage-party docker-push-triage-party clean-triage-party
+
+.PHONY: release-local-triage-party
+release-local-triage-party: docker-build-triage-party clean-triage-party ## Release the triage party image for local use only
+
+.PHONY: checkout-triage-party
+checkout-triage-party:
+	@if [ -z "${TRIAGE_PARTY_VERSION}" ]; then echo "TRIAGE_PARTY_VERSION is not set"; exit 1; fi
+	@if [ -d "$(TRIAGE_PARTY_TMP_DIR)" ]; then \
+		echo "$(TRIAGE_PARTY_TMP_DIR) exists, skipping clone"; \
+	else \
+		git clone "https://github.com/google/triage-party.git" "$(TRIAGE_PARTY_TMP_DIR)"; \
+		cd "$(TRIAGE_PARTY_TMP_DIR)"; \
+		git checkout "$(TRIAGE_PARTY_VERSION)"; \
+		git apply "$(ROOT_DIR)/$(TRIAGE_PARTY_DIR)/triage-improvements.patch"; \
+	fi
+	@cd "$(ROOT_DIR)/$(TRIAGE_PARTY_TMP_DIR)"; \
+	if [ "$$(git describe --tag 2> /dev/null)" != "$(TRIAGE_PARTY_VERSION)" ]; then \
+		echo "ERROR: checked out version $$(git describe --tag 2> /dev/null) does not match expected version $(TRIAGE_PARTY_VERSION)"; \
+		exit 1; \
+	fi
+
+.PHONY: docker-build-triage-party
+docker-build-triage-party: checkout-triage-party
+	@if [ -z "${TRIAGE_PARTY_VERSION}" ]; then echo "TRIAGE_PARTY_VERSION is not set"; exit 1; fi
+	cd $(TRIAGE_PARTY_TMP_DIR) && \
+	docker buildx build --platform linux/amd64 -t $(TRIAGE_PARTY_CONTROLLER_IMG):$(TRIAGE_PARTY_VERSION) .
```

Comment:
Q: Do we have to hard-code this to linux? (I assume it just magically works on Mac anyway? then it should be good enough)

Reply:
But this linux image then just works on Mac because of Mac-magic, correct?

Reply:
yes, correct